ERROR 2013-11-20 05:42:57,651 speed_logging_utils 4336 140318330132256 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-20 05:42:57,951 speed_logging_utils 4336 140318330132256 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-20 05:42:58,060 speed_logging_utils 4336 140318330132256 browser speed log HTTP request JSON parse failed
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 93, in read_browser_speed_data
    speed_data = json.loads(request.body)
  File "/usr/lib64/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 360, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 378, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
ERROR 2013-11-20 05:42:58,235 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:58,370 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:58,501 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'django_begins'
ERROR 2013-11-20 05:42:58,629 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-20 05:42:58,757 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:58,889 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (django_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:59,022 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:59,156 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:42:59,289 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-20 05:42:59,415 speed_logging_utils 4336 140318330132256 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_ends'
ERROR 2013-11-20 05:45:01,510 speed_logging_utils 4754 140180066019104 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-20 05:45:01,818 speed_logging_utils 4754 140180066019104 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-20 05:45:01,927 speed_logging_utils 4754 140180066019104 browser speed log HTTP request JSON parse failed
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 93, in read_browser_speed_data
    speed_data = json.loads(request.body)
  File "/usr/lib64/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 360, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 378, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
ERROR 2013-11-20 05:45:02,100 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:02,232 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:02,363 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'django_begins'
ERROR 2013-11-20 05:45:02,491 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-20 05:45:02,614 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:02,744 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (django_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:02,887 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:03,023 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-20 05:45:03,158 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-20 05:45:03,285 speed_logging_utils 4754 140180066019104 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_ends'
